src/styles/foundation/_focus-ring.scss

```diff
@@ -1,8 +1,9 @@
 /// Sets the focus ring for an interactive element
 /// @param {String} $size - The size of the border radius on the focus ring.
+/// @param {BorderStyle} $style - The style of the border on the focus ring.
 ///
 
-@mixin focus-ring($size: 'base') {
+@mixin focus-ring($size: 'base', $style: solid) {
```

Comment:
Let's keep the focus ring consistent across all elements, that way it's totally obvious to merchants any time an element is in the focused state. Having one-off variations like `dashed` is counter to this consistency. I think in this case, the focus ring can wrap outside the default state dashed border with a `1px` offset.